dask_kubernetes/operator/core.py

```diff
@@ -0,0 +1,52 @@
+import subprocess
+
+from distributed.deploy import Cluster
+from distributed.core import rpc
+
+# import kopf
+import kubernetes
+
+from dask_kubernetes.utils import (
+    #     namespace_default,
+    get_external_address_for_scheduler_service,
+    #     check_dependency,
+)
+
+
+class KubeCluster2(Cluster):
+    """Launch a Dask Cluster on Kubernetes using the Operator"""
+
+    def __init__(self, name, loop=None, asynchronous=False, **kwargs):
+        self.name = name
+        self.core_api = None
+        self.scheduler_comm = None
+
+        status = subprocess.run(
+            ["kopf", "run", ".daskcluster.py"],
+            capture_output=True,
+            encoding="utf-8",
+        )
```

Comment:
We shouldn't be running the operator here. That should be running already on our cluster independently of what Dask clusters we create.